INSTALLATION.md

```diff
@@ -8,71 +8,57 @@ Required dependencies:
   - Clang >= 6
   - Apple Clang >= 9.1
 - [CMake](http://www.cmake.org) >= 3.11
+- [Arrow](https://arrow.apache.org/)
 
 Optional dependencies:
 
 - [libpcap](http://www.tcpdump.org)
 - [gperftools](http://code.google.com/p/google-perftools)
 - [Doxygen](http://www.doxygen.org)
 - [Pandoc](https://github.com/jgm/pandoc)
-- [Arrow](https://arrow.apache.org/)
 
-### Source Build
+## Source Build
 
 Building VAST involves the following steps:
 
-```sh
-git submodule update --recursive --init
-./configure
-cmake --build build
-cmake --build build --target test
-cmake --build build --target install
-```
-
-The `configure` script is a small wrapper that passes build-related variables
-to CMake. For example, to use [Ninja](https://ninja-build.org) as build
-generator, add `--generator=Ninja` to the command line. Passing `--help` shows
-all available options.
-
-The `doc` target builds the API documentation locally:
-
-```sh
-cmake --build build --target doc
-```
-
-## Docker
-
-The source ships with the convenience script `scripts/docker-build`, which will
-create the Docker images and save them as `tar.gz` archives (when invoked
-without arguments).
+1. Clone the repository recursively:
+  ```sh
+  git clone --recursive https://github.com/tenzir/vast
+  ```
+2. Configure the build, either with our `configure` wrapper script or manually
+   with CMake:
+  ```sh
+  cd vast
+  ./configure
+  ```
+3. Build the executable:
+  ```sh
+  cmake --build build --target all
+  ```
+4. Run the tests to verify everything works as expected:
+  ```sh
+  cmake --build build --target unit-tests
```

Comment:
The unit-tests target is named `test`, and the integration-tests target is named `integration`.